test/12_nullablearray.jl

```diff
@@ -965,4 +965,66 @@ for isordered in (false, true)
     end
 end
 
+
+# Tests of vcat of NullableCategoricalArray
+
+for (CA,A) in ((:NullableCategoricalArray,:NullableArray),)
```

Comment:
Oh, and BTW you don't need `@eval` here: `CA` and `A` can simply be types instead of symbols, and you don't need `$`.